src/services/CertificateGenerationService.ts

```diff
@@ -1178,6 +1252,56 @@ class CertificateGenerationService {
     return adrTestTypeIds.includes(testType.testTypeId);
   }
 
+    /**
+     * Returns true if testType is iva and false if not
+     * @param testType - testType which is tested
```

Comment:
can this be reworded and also it's missing the return so could make it something like
checks the whether the test type provided is iva
param testType - test type being tested
returns boolean true if test type is iva otherwise false